spec/mongo/server/push_monitor/connection_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe Mongo::Server::PushMonitor::Connection do
+  describe '#check_document' do
+    it 'returns hello document with API version' do
+      meta = Mongo::Server::AppMetadata.new({
```

Comment:
`{}` may be omitted for the last (or only) argument of a method which is a hash.